### ISSUE TITLE
management/spawnas not obfuscated

### ISSUE THREAD

**Issue by dmittakarin8** [cobbr/ObfuscatedEmpire#13]
## Empire Version
2.0

## OS Information (Linux flavor, Python version)
Linux Kali 4.9.0-kali-4-amd64
Python 2.7.13

## Expected behavior and description of the error, including any actions taken immediately prior to the error. The more detail the better.
-----------------
Expected behavior
-----------------
  - Execute management/spawnas obfuscated
-----------------
Error 
-----------------
  - spawnas not getting obfuscated
-----------------
Actions taken
-----------------
- Establish initial low privilege agent - (Obfuscated without issue)
   - ObfuscateCommand = Token\Comment\1,Token\Command\2,Token\Argument\2,Token\Member\4,Token\Variable\1,Token\Variable\1,Token\Type\2,Launcher\Clip+\6
- set global obfuscate True
- set global obfuscate_command = Token\Comment\1,Token\Command\2,Token\Argument\2,Token\Member\4,Token\Variable\1,Token\Variable\1,Token\Type\2,Launcher\Clip+\6
- interact with agent
- usemodule management/spawnas
   - set UserName
   - set Domain
   - set Password
   - set Listener
   - unset Proxy
   - unset ProxyCreds
- Execute
- Error
   - "error running command: The command parameter was already specified"
- set global obfuscate_command (without launcher) = Token\Comment\1,Token\Command\2,Token\Argument\2,Token\Member\4,Token\Variable\1,Token\Variable\1,Token\Type\2
- interact with agent
- usemodule management/spawnas
- execute
- review debug.bat in c:\Users\Public
- Command is NOT obfuscated

## Screenshot of error, embedded text output, or Pastebin link to the error
There are two errors trying to obfuscate the management/spawnas module
1.  With a launcher set in the obfuscate_command the error returned after executing the module is:
     "error running command: The command parameter was already specified"
2.  Removing the launcher from the obfuscate_command executes the spawnas module, but spawnas is NOT getting obfuscated.

## Any additional information
In summary, I cannot get the spawnas module to run obfuscated.

**Comment by dmittakarin8** [comment]
Performing a reset seems to have resolved my issue with getting spawnas to run obfuscated.  However spawnas does not get executed when obfuscated.  I will open a different issue.
